samples/drivers/adc/boards/xmc45_relax_kit.overlay

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2022 Andriy Gelman <andriy.gelman@gmail.com>
+ */
+
+#include <zephyr/dt-bindings/adc/adc.h>
+
+/ {
+	zephyr,user {
+		io-channels = <&adc0 0>;
+	};
+};
+
+&adc0 {
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	channel@0 {
+		reg = <0>;
+		zephyr,acquisition-time = <ADC_ACQ_TIME_DEFAULT>;
+		zephyr,gain = "ADC_GAIN_1";
+		zephyr,reference = "ADC_REF_INTERNAL";
+		zephyr,resolution = <12>;
+	};
+	status = "okay";
+};
```

Comment:
@talih0 Is this already tested with 4500 relax kit?

Reply:
@parthitce, yes, here's a screenshot from this sample where I touch P14.0 to 3.3V and then to GND:

![image](https://user-images.githubusercontent.com/17433307/207335872-0b914eb2-54f5-4d96-b2e8-71631afc1aea.png)